rootex/core/renderer/constant_buffer.h

```diff
@@ -40,19 +48,26 @@ struct SpotLightInfo
 	float attQuad = 0.0075f;
 	Vector3 lightPos = { 0.0f, 0.0f, 0.0f };
 	float range = 10;
+	/// direction of axis of light cone
 	Vector3 direction;
+	/// increasing spot increases the angular attenuation wrt axis
```

Comment:
Increasing, and now review the rest of the changes. Fix capitalization issues in doc strings.